mrbelvedereci/testresults/utils.py

```diff
@@ -8,7 +9,7 @@ def find_buildflow(request, build_id, flow):
     build = get_object_or_404(Build, id=build_id)
 
     if not build.plan.public and not request.user.is_staff:
-        raise Http404()
+        raise Http403()
```

Comment:
`Http403` doesn't exist, should be `HttpResponseForbidden`. Sorry if my previous comment was unclear!